test/recipes/70-test_tls13messages.t

```diff
@@ -134,6 +134,11 @@ $ENV{CTLOG_FILE} = srctop_file("test", "ct", "log_list.conf");
     [0,0,0]
 );
 
+use constant {
+    PURE_SIGALGS => 0,
+    COMPAT_SIGALGS => 1
+};
+
```

Comment:
Ah, good point.
Fixed, and squashed the whole stack down since it was getting a bit silly.